lib/access_import/access_import.rb

```diff
@@ -0,0 +1,34 @@
+require_all 'lib/access_import'
```

Comment:
So you mean if AccessImport'd be a module instead of a class it would auto load it?